### PR TITLE
Enhance versioning support

### PR DESCRIPTION
**Pull request by katya-dovgalets**
A partial solution for https://github.com/michael-simons/neo4j-migrations/issues/159

### CODE REVIEW

File: neo4j-migrations-core/src/main/java/ac/simons/neo4j/migrations/core/MigrationVersion.java [hunk 1]
```diff
@@ -28,10 +28,10 @@
  */
 public final class MigrationVersion {
 
-	private static final String BASLINE_VALUE = "BASELINE";
-	private static final MigrationVersion BASELINE = new MigrationVersion(BASLINE_VALUE, null);
+	private static final String BASELINE_VALUE = "BASELINE";
```
Comment (michael-simons) [inline neo4j-migrations-core/src/main/java/ac/simons/neo4j/migrations/core/MigrationVersion.java:31]: Nice catch!! 👍

**Review by michael-simons — CHANGES_REQUESTED** [review]
That's a great PR, thank you!

I have only to complain what check style complains as well:
Please run `mvn checkstyle:check@validate` and fix the small number of things, please (mainly license header).

I will merge this and happily release it.

Wrt repeatable versions: Hardly at the moment
Wrt undoable: I have to think about it a bit.

**Comment by michael-simons** [comment]
Hm, why did you close it?

**Comment by katya-dovgalets** [comment]
Reopened 
PR was closed automatically after force push in my branch.

**Comment by michael-simons** [comment]
Thank you very much, Kateryna! Having a first contributor here makes me really happy.

**Comment by michael-simons** [comment]
@all-contributors add @katya-dovgalets for code

**Pull request merged**